document/FileFormat.mdk

```diff
@@ -2672,16 +2675,20 @@ This can be used, for example, to designate the target of a jump instruction, or
 * `"declaration"`: The location introduces into the program a name which denotes an entity such as a variable, function, template, _etc_.
 
 * `"functionEnter"`: This location is an entry point to a function or method.
+    Every `annotatedCodeLocation` whose `kind` property is `"functionEnter"` shall be paired with a
+    subsequent `annotatedCodeLocation` whose `kind` property is `"functionExit"` and which refers to the same function,
+    unless the `codeFlow` in which the call occurs terminates before a function exit point is reached. 
 
-* `"functionExit"`: This location is an exit point from a function or method (for example, a return statement).
+* `"functionExit"`: This location represents the actual exit point from the function.
```

Comment:
>the actual exit point [](start = 45, length = 21)

I'd simply say the functionExit represents the exit from a function. If you say 'actual' or 'exit point', this suggests there is an explicit location, which is not necessarily true. You might add a follow-on statement like:

A function exit may have no location, be associated with the closing brace of a function, or represented by the final statement. A function exit may be preceded by a functionReturn.